src/Views/ProgressView.vala

```diff
@@ -15,7 +15,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-public class ProgressView : AbstractInstallerView {
+public class ProgressView : Gtk.Box {
```

Comment:
Why is this being changed away from AbstractInstallerView?

Reply:
Because it's the only pane that isn't a side-by-side layout and the abstract view now enforces the side-by-side layout